docs/ii-spec.mdx

```diff
@@ -468,22 +477,58 @@ This call is used to supply a solution to the captcha challenge returned from `i
 
 Replaces the `metadata` map of the given authn_method.
 
 #### The `authn_method_security_settings_replace` method
+
 **Authorization**: This request must be sent to the canister with `caller` that is the self-authenticating id derived from any of the public keys of devices associated with the user before this call.
 
 Replaces the `authn_method_security_settings_replace` map of the given authn_method. This method is split from `authn_method_metadata_replace` in order to be able to introduce different security requirements for security relevant changes to authn_methods while not impeding non-critical changes (such as e.g. changing the authn_method alias).
 
 #### The `identity_metadata_replace` method
+
 **Authorization**: This request must be sent to the canister with `caller` that is the self-authenticating id derived from any of the public keys of devices associated with the user before this call.
 
 Replaces the `metadata` map associated with the given identity.
 
+### Account Management
+
+Internet Identity supports Accounts. Accounts are subordinate entities to an Identity and allow a user to appear to any given dApp as a completely different user/principal while using the same Identity and authentication methods. Accounts are separated per origin - in practice this means that you will have one set of accounts for each dapp. They also each have a unique account number assigned to them.
+
+There are two flavors of Accounts:
+
+- Primary Accounts are not named and always available. They show up as the same principal to any given dApp as your Identity did prior to the introduction of Accounts.
+- Secondary Accounts have custom names and are currently limited to 500 total, across origins. They show up as a new and unique principal to a dApp.
+
+Accounts can be renamed. Renaming a Primary Account turns them into a Secondary Account. Accounts can not currently be deleted.
+
+The following endpoints relate to Account Management:
+
+#### The `create_account` method
+
+**Authorization**: This request must be sent to the canister with `caller` that is the self-authenticating id derived from any of the public keys of devices associated with the user before this call.
+
+The `create_account` method causes the Internet Identity Service backend to create a new Secondary account for the user for a specific origin. This is counted against the total accounts limit.
+
+#### The `update_account` method
+
+**Authorization**: This request must be sent to the canister with `caller` that is the self-authenticating id derived from any of the public keys of devices associated with the user before this call.
+
+The `update_account` method causes the Internet Identity Service backend to update an existing account for the user. Currently, only account names can be updated. Updating the name of a Primary account turns it into a Secondary account and is counted against the total accounts limit. It has no influence on the principal of the account.
```

Comment:
Ditto about "turning it into a secondary account". I would just mention that it will start counting against the accounts limit. I wouldn't call it either "total accounts limit", just "accounts limit".

Reply:
The part of `"It has no influence on the principal of the account."` is a good point. I would put it in a separate paragraph and be more explicit: calls to "update_account" have no influence on the principal of the account.